src/App.js

```diff
@@ -94,6 +94,8 @@ class App extends Component {
             <PublicRouteWithLegacyFallback exact path="/forgot-password" />
             <RedirectWithQuery exact from="/ingresa" to="/login?lang=es" />
             <RedirectWithQuery exact from="/registrate" to="/signup?lang=es" />
+            <RedirectWithQuery exact from="/ingresa/cambiar-clave" to="/forgot-password?lang=es" />
+            <RedirectWithQuery exact from="/login/reset-password" to="/forgot-password?lang=en" />
```

Comment:
We have talked with @cbernat and decided using `/login/reset-password` as official URL and redirect `/forgot-password` to `/login/reset-password`.